config/prov/prov_test.yaml

```diff
@@ -217,15 +219,15 @@ k8s_cortx_deploy:
         mem: '1Gi'
         cpu: '250m'
       limits:
-        mem: '3Gi'
+        mem: '2Gi'
```

Comment:
Please add reference for these values.

Reply:
Please refer the solution.example.yaml file in K8s repo.https://github.com/Seagate/cortx-k8s/blob/v0.9.0/k8_cortx_cloud/solution.example.yaml